keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/BuildMetadata.kt

```diff
@@ -1,9 +1,14 @@
 package com.netflix.spinnaker.keel.api.artifacts
 
 /**
- * todo eb: other information should go here, like a link to the jenkins build. But that needs to be done
- * in a scalable way. For now, this is just a minimal container for information we can parse from the version.
+ * The build metadata of an artifact.
  */
 data class BuildMetadata(
```

Comment:
@erikmunson I'm planning to extend `BuildMetadata` and `GitMetadata` below with those fields. I'm sure we will not be showing all of them -- but please let me know if you think something is missing here. The `uid` in this class is the boost build id we are using to fetch the logs -- it will be the path for linking the logs, potentially in the bottom part of the tab when we are showing CI information for the artifact.